doc/source/getting_started/accessing_models.rst

```diff
@@ -0,0 +1,109 @@
+.. _accessing_models:
+
+================
+Accessing Models
+================
+
+
+.. note:: Suppose you have started the Xinference server endpoint at ``http://0.0.0.0:8001``. 
+
+Please refer to :ref:`Launching Models <launching_models>` guide for get your model running.
+
+Using Clients
+=============
+
+
+LLM Model
+---------
+When the Ability of the LLM model includes "chat", we can converse with it using the model's chat interface:
+
+.. code-block:: python
+
+   from xinference.client import RESTfulClient
+   client = RESTfulClient("http://0.0.0.0:8001")
+   model = client.get_model(model_uid)
+
+   chat_history = []
+   prompt = "What is the largest animal?"
+   model.chat(
+       prompt,
+       chat_history,
+       generate_config={"max_tokens": 1024}
+   )
+
+The response will look like:
+
+.. code-block:: json
+
+   {
```

Comment:
I generated another response that includes the usage.

```json
{
   "id":"chat3626defa-55f2-11ee-8f6c-0a28cc89f433",
   "object":"chat.completion",
   "created":1695020931,
   "model":"f543d078-55f1-11ee-8fc0-0a28cc89f433",
   "choices":[
      {
         "index":0,
         "message":{
            "role":"assistant",
            "content":"The largest animal in the world is the blue whale. It can grow up to 100 feet long and weigh up to 200 tons. It is the largest mammal on Earth and the largest living thing that ever existed. Blue whales are found in all of the world's oceans and are the only animals that can communicate with each other through a series of clicks and whistles. They are also the fastest animals in the world, able to swim at speeds of up to 35 miles per hour."
         },
         "finish_reason":"stop"
      }
   ],
   "usage":{
      "prompt_tokens":48,
      "completion_tokens":117,
      "total_tokens":165
   }
}
```

Reply:
Would you please make a change to this file so I can commit it on the webpage